src/main/java/seedu/duke/ui/UI.java

```diff
@@ -37,6 +39,40 @@ public static String formatExpenditure(String description, double price, String
                 MessageConstants.NEWLINE;
     }
 
+    /**
+     * Returns a string based on the details of the Entry object and entryID entered.
+     *
+     * @param entry   entry Object to be formatted
+     * @param entryID ID of the entry Object
+     * @return String of details about the entry
+     */
+    private String formatViewEntries(Entry entry, int entryID) {
+        String description = entry.getDescription();
+        double price = entry.getAmount();
+        String category = entry.getCategoryString();
+        return "<" + Integer.toString(entryID) + ">: " + description +
+                " (" + category + ") - $" + formatPrice(price);
+    }
+
+    /**
+     * Combines all individual entry strings into a list of entries and prints the list, along with an acknowledgement
+     * message.
+     *
+     * @param entryList List of entry objects with the user-specified length
+     * @param category String denoting the category requested by the user
+     */
+    public void printEntriesToBeViewed(List<Entry> entryList, String category) {
+        StringBuilder finalString = new StringBuilder();
+        finalString.append("These are the latest " + entryList.size() + " entries from the " + category +
```

Comment:
Consider utilising `System.lineSeparator()` for the newline instead of `\n`. Alternatively you can just use `MessageConstants.NEWLINE`.